frontend/simple/views/components/Modals/index.js

```diff
@@ -0,0 +1 @@
+export { default as ModalForm } from './ModalForm.vue'
```

Comment:
What is the purpose of this 1-line file?

Reply:
Soon I see we will have more type of modals, at least a `ModalDefault` or `ModalConfirm`, or whatever the type of modal. This is a [barrel pattern](https://basarat.gitbooks.io/typescript/docs/tips/barrel.html)